jxl_simd/src/aarch64/neon.rs

```diff
@@ -122,6 +122,140 @@ impl F32SimdVec for F32VecNeon {
         unsafe { vst1q_f32(mem.as_mut_ptr(), self.0) }
     }
 
+    #[inline(always)]
+    fn store_interleaved_2(a: Self, b: Self, base: &mut [f32], offset: usize) {
+        assert!(base.len() >= offset + 2 * Self::LEN);
```

Comment:
If we keep the offset, we also need to check that the computation does not overflow.